_data/students.yml

```diff
@@ -145,7 +145,16 @@
   country: Singapore
   github: ChenNupde
   image: Nuode.jpg
-  
+
+  name: Naveen Rajan
```

Comment:
You are already in students.

Reply:
Well I just added My Self

Reply:
https://github.com/naveentherocker/gci17.fossasia.org/blob/43c31a59e61713a2d17274c468360863d672c802/_data/students.yml#L27

Reply:
In your fork, make updates here.

Reply:
Now it's fine